### PR TITLE
feat:implement responsive learn-more section

### PR DESCRIPTION
**Pull request by MabelM1**

### CODE REVIEW

File: blocks/additional-info/__link/additional-info__link.css [hunk 1]
```diff
@@ -3,6 +3,13 @@
   line-height: 19px;
 }
 
+@media only screen and (max-width: 320px) {
```
Comment (tantany) [inline blocks/additional-info/__link/additional-info__link.css:6]: I don't have where to write it but, the additional title in 320px screen is 28px and should be 20px.
Also, why in additional info the title block is unique to additional info? why not use content title?

File: blocks/additional-info/__subtitle/additional-info__subtitle.css [hunk 2]
```diff
@@ -15,5 +15,6 @@
   .additional-info__subtitle {
     font-size: 18px;
     line-height: 18px;
+    margin: 22px 0 0;
```
Comment (tantany) [inline blocks/additional-info/__subtitle/additional-info__subtitle.css:18]: In 320px screen the size of the subtitle is 12px

File: blocks/arrow-icon/arrow-icon.css [hunk 3]
```diff
@@ -8,3 +8,11 @@
 .arrow-icon:hover {
   background: url("./yellow-arrow.png") center center no-repeat;
 }
+
+@media only screen and  (max-width: 320px) {
+  .arrow-icon {
+    font-size: 14px;
```
Comment (tantany) [inline blocks/arrow-icon/arrow-icon.css:14]: The icon of the arrow is aligned to the upper row of the link and not the lower row

File: blocks/arrow-icon/arrow-icon.css [hunk 4]
```diff
@@ -8,3 +8,11 @@
...
+    width: 26px;
+    height: 22px;
```
Comment (tantany) [inline blocks/arrow-icon/arrow-icon.css:16]: The arrow is too big compared to the figma design

File: blocks/additional-info/__list-item/additional-info__list-item.css [hunk 5]
```diff
@@ -1,11 +1,11 @@
 .additional-info__list-item {
-  margin-bottom: 20px;
+  margin-bottom: 16px;
```
Comment (tantany) [inline blocks/additional-info/__list-item/additional-info__list-item.css:2]: You should add "last-of-type", you have extra margin on the second list item

Comment (tantany) [inline blocks/additional-info/__list-item/additional-info__list-item.css:2]: margin bottom should be 18px for 320px

File: blocks/additional-info/__link/additional-info__link.css [hunk 6]
```diff
@@ -3,6 +3,13 @@
...
+  .additional-info__link {
+    font-size: 14px;
```
Comment (tantany) [inline blocks/additional-info/__link/additional-info__link.css:8]: You should specify font-weight everywhere. font-weight 500 looks different the the normal one.

File: blocks/additional-info/__link/additional-info__link.css [hunk 7]
```diff
@@ -3,6 +3,13 @@
...
+    line-height: 22px;
+  }
+}
+
 @media only screen and (min-width: 769px) and (max-width: 1024px) {
```
[the remark this answers is not in the document]
Comment (MabelM1) [reply blocks/additional-info/__link/additional-info__link.css:13]: I know

File: blocks/additional-info/__link/additional-info__link.css (hunk 1 above)
Comment (MabelM1) [reply blocks/additional-info/__link/additional-info__link.css:6]: switched

File: blocks/additional-info/__subtitle/additional-info__subtitle.css (hunk 2 above)
Comment (MabelM1) [reply blocks/additional-info/__subtitle/additional-info__subtitle.css:18]: switched to content_subtitle

File: blocks/arrow-icon/arrow-icon.css (hunk 3 above)
Comment (MabelM1) [reply blocks/arrow-icon/arrow-icon.css:14]: updated

File: blocks/arrow-icon/arrow-icon.css (hunk 4 above)
Comment (MabelM1) [reply blocks/arrow-icon/arrow-icon.css:16]: fixed

File: blocks/additional-info/__list-item/additional-info__list-item.css (hunk 5 above)
Comment (MabelM1) [reply blocks/additional-info/__list-item/additional-info__list-item.css:2]: done

Comment (MabelM1) [reply blocks/additional-info/__list-item/additional-info__list-item.css:2]: done

**Pull request merged**